src/hooks/portfolio/useFormatDisplayDeFiPositionsData.ts

```diff
@@ -0,0 +1,64 @@
+import { useMemo } from 'react';
+import type { DefiPosition } from '@/types/jumper-backend';
+import type { MinimalDeFiPosition } from 'src/types/defi';
+
+// @Note: This is a temporary solution to convert the data to the MinimalDeFiPosition type.
```

Comment:
could you create a ticket, and describe a bit the issue, and reference it in that note? I don't have full context but that sounds important.

Reply:
Perfect, thanks, that should be referenced in the note